plugins/zoom/plugin.spec.yaml

```diff
@@ -146,87 +166,100 @@ types:
       enum:
       - 1
       - 0
+      example: 1
     dept:
       title: Department
       description: Department of user
       type: string
       required: false
+      example: example department
     created_at:
       title: Created At
       description: Creation datetime of user
       type: string
       required: false
+      example: 2023-06-22T12:26:02Z
     last_login_time:
       title: Last Login Time
       description: Last login datetime of user
       type: string
       required: false
+      example: 2023-06-21T13:41:14Z
     host_key:
       title: Host Key
       description: User's host key
       type: string
       required: false
+      example: 123321
     jid:
       title: JID
       description: JID of user
       type: string
       required: false
+      example: T9ti3NBxR42swGKrqABGig@xmpp.zoom.us
     group_ids:
       title: Web Group IDs
       description: IDs of the web groups user belongs to
       type: "[]string"
       required: false
+      example: ["t-_-d56CSWG-7BF15LLrOw", "t-_-d56CSWG-7BF15LLrow"]
     im_group_ids:
       title: IM Group IDs
       description: IM IDs of the groups user belongs to
       type: "[]string"
       required: false
+      example: ["t-_-d56CSWG-7BF15LLrOw", "t-_-d56CSWG-7BF15LLrow"]
     account_id:
       title: Account ID
       description: Account ID of user
       type: string
       required: false
+      example: T9ti3NBxR42swGKrqABGig
     language:
       title: Language
       description: Language of user
       type: string
       required: false
+      example: en-US
     phone_country:
       title: Phone Country
       description: Phone country of user
       type: string
       required: false
+      example: US
     phone_number:
       title: Phone Number
       description: Phone number of user
       type: string
       required: false
+      example: 555-0100
     status:
       title: Status
       description: Status of user
       type: string
       required: false
+      example: active
 actions:
   get_user:
     title: Get User
-    description: Get user in Zoom account
+    description: This action is used to get user in Zoom account
```

Comment:
hey so for now dont worry about adding the prefix here. Keep the description the same, we're gonna get the tooling to handle adding the prefix just for the help.md. Best to keep these descriptions succinct like what it was before this change